src/pages/resultsView/enrichments/EnrichmentsUtil.tsx

```diff
@@ -81,10 +81,10 @@ export function getAlterationRowData(alterationEnrichments: AlterationEnrichment
             hugoGeneSymbol: alterationEnrichment.hugoGeneSymbol,
             entrezGeneId: alterationEnrichment.entrezGeneId,
             cytoband: alterationEnrichment.cytoband, 
-            alteredCount: alterationEnrichment.alteredCount,
-            alteredPercentage: alterationEnrichment.alteredCount / totalAltered * 100,
-            unalteredCount: alterationEnrichment.unalteredCount,
-            unalteredPercentage: alterationEnrichment.unalteredCount / totalUnaltered * 100,
+            alteredCount: alterationEnrichment.set1CountSummary.alteredCount,
+            alteredPercentage: alterationEnrichment.set1CountSummary.alteredCount / totalAltered * 100,
+            unalteredCount: alterationEnrichment.set2CountSummary.alteredCount,
+            unalteredPercentage: alterationEnrichment.set2CountSummary.alteredCount / totalUnaltered * 100,
```

Comment:
@onursumer @zhx828 @averyniceday should the denominator be ..CountSummary.profiledCount? or total count as it is now?

Reply:
profiledCount sounds right to me. This is similar to what we have in study view about sequenced vs all?

Reply:
I think it should be profiledCount. Note that different genes may have different profiledCount. We should really take the count of samples/patients that are profiled for both genes and it should probably be returned in the alterationEnrichment result.

How does the backend do the counting?

Reply:
@averyniceday can you answer this?

Reply:
@ao508 can you comment on this?